platform/jewel/ide-laf-bridge/src/main/kotlin/org/jetbrains/jewel/bridge/ToolWindowExtensions.kt

```diff
@@ -17,6 +19,7 @@ public fun ToolWindow.addComposeTab(
     @TabTitle tabDisplayName: String? = null,
     isLockable: Boolean = true,
     isCloseable: Boolean = false,
+    focusOnClickInside: Boolean = true,
```

Comment:
I reckon the default for toolwindows should be true, to match Swing's behaviour. Wdyt?